.github/workflows/ci.yml

```diff
@@ -32,28 +32,28 @@ jobs:
 
       - name: Apt install
         run: cat Aptfile | sudo xargs apt-get install
+        
+      - name: Install poetry
+        uses: snok/install-poetry@v1
+        with:
+          version: 1.5.1
+          virtualenvs-create: true
+          virtualenvs-in-project: true
```

Comment:
For local dev we set
```
ENV  \
  # poetry:
  POETRY_VERSION=1.5.1 \
  POETRY_VIRTUALENVS_CREATE=false \
  POETRY_CACHE_DIR='/tmp/cache/poetry'
```
below.

Any reason to create the virtualenvs for CI?

Reply:
After changing to:

```
          virtualenvs-create: false
          virtualenvs-in-project: false
```

CI failed:

```
Run poetry install --no-interaction
  poetry install --no-interaction
  shell: /usr/bin/bash -e {0}
  env:
    VENV: .venv/bin/activate
    pythonLocation: /opt/hostedtoolcache/Python/3.9.13/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.9.13/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.13/x64
    Python[2](https://github.com/mitodl/odl-video-service/actions/runs/5379641451/jobs/9761161449#step:7:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/mitodl/odl-video-service/actions/runs/5379641451/jobs/9761161449#step:7:3).9.13/x6[4](https://github.com/mitodl/odl-video-service/actions/runs/5379641451/jobs/9761161449#step:7:4)
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.13/x[6](https://github.com/mitodl/odl-video-service/actions/runs/5379641451/jobs/9761161449#step:7:6)4
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.[9](https://github.com/mitodl/odl-video-service/actions/runs/5379641451/jobs/9761161449#step:7:9).13/x64/lib
Skipping virtualenv creation, as specified in config file.

Invalid version '1.1build1' on package distro-info
Error: Process completed with exit code 1.
```

partially reverted to:
```
          virtualenvs-create: true
          virtualenvs-in-project: false
```

and got a different error slightly later:

```
[TEST SUITE] pytest
Creating virtualenv odl-video-service-tH1iv-yH-py3.10 in /src/.cache/pypoetry/virtualenvs
usage: virtualenv [--version] [--with-traceback] [-v | -q] [--read-only-app-data] [--app-data APP_DATA] [--reset-app-data] [--upgrade-embed-wheels] [--discovery {builtin}] [-p py] [--try-first-with py_exe]
                  [--creator {builtin,cpython3-posix,venv}] [--seeder {app-data,pip}] [--no-seed] [--activators comma_sep_list] [--clear] [--no-vcs-ignore] [--system-site-packages] [--symlinks | --copies] [--no-download | --download]
                  [--extra-search-dir d [d ...]] [--pip version] [--setuptools version] [--wheel version] [--no-pip] [--no-setuptools] [--no-wheel] [--no-periodic-update] [--symlink-app-data] [--prompt prompt] [-h]
                  dest
virtualenv: error: argument dest: the destination . is not write-able at /
```

Reply:
@ChristopherChudzicki I think Matt's work here shows us that the test suite as it currently exists relies on the existence of an actually fully created and initialized venv.

Given that I don't see a problem with him leaving the createvirtualenv values set they way there were.